config_test_imdb.yml

```diff
@@ -0,0 +1,21 @@
+experiment_name: test_imdb
+n_repeats: 1
+scenario_params_list:
+ - dataset_name:
+    - 'imdb'
+   partners_count:
+    - 3
+   amounts_per_partner: 
+    - [0.2, 0.5, 0.3]
+   samples_split_option:
+    - ['advanced', [[1, 'shared'], [1, 'shared'], [1, 'specific']]]
```

Comment:
Hmmm, the experiment you configured is quite particular:

- You want 2 partners to share samples with 1 label, and a 3rd partner to have samples with another label
- But there are only 2 different labels in this dataset, let's call them `true` and `false` to make it simple here
- This means you have 2 partners with only `true` data samples, and 1 with only `false` data sample
- Learning a model in that configuration is not an obvious thing

I'd strongly suggest that you pick a simpler configuration (a `['basic', 'random'] for example) for validating your implementation.